atomicfu-maven-plugin/build.gradle.kts

```
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+plugins {
+    `maven-publish`
+    id("kotlin-jvm-conventions")
+}
+
+dependencies {
+    compileOnly(libs.kotlin.stdlib)
+    api(project(":atomicfu-transformer"))
+    api(libs.bundles.maven)
+}
+
+val outputDir = tasks.compileKotlin.get().destinationDirectory
+
+publishing.publications {
+    named<MavenPublication>("maven").configure {
+        pom.configureMavenPluginPomAttributes(project, outputDir.get().asFile.path)
+        pom.packaging = "maven-plugin"
+    }
+}
+
+val mavenUserHome = System.getProperty("maven.user.home")
+val mavenRepoLocal = System.getProperty("maven.repo.local")
```

```suggestion
val mavenUserHome: String? = System.getProperty("maven.user.home")
val mavenRepoLocal: String? = System.getProperty("maven.repo.local")
```

Let's specify the types for clarity and to avoid the "Declaration has type inferred from a platform call..." warning.

---

fixed